tests/php/LeftAndMainTest.php

```diff
@@ -135,65 +128,6 @@ public function testLeftAndMainSubclasses()
         $this->assertMatchesRegularExpression('/<body[^>]*>/i', $response->getBody(), "$link should contain <body> tag");
     }
 
-    public function testCanView()
```

Comment:
This was testing the wrong thing - it was trying to test `canView()` checks by seeing what's in the menu. But the menu actually _shouldn't_ contain `CMSProfileController` so this test was passing when it should have failed.

Moved canView check to the new test class, and now actually testing canView instead of the menu.

The menu itself is already tested in this class.